src/steps/generic.rs

```diff
@@ -433,11 +433,25 @@ pub fn run_vscodium_extensions_update(ctx: &ExecutionContext) -> Result<()> {
         .lines()
         .next()
     {
-        Some(item) => Version::parse(item).map_err(std::convert::Into::into),
+        Some(item) => {
+            // Strip leading zeroes because `semver` does not allow them, but VSCodium uses them sometimes
```

Comment:
Is this exclusive to VSCodium or is it also present in VSCode as well?

Reply:
This is not present in VSCode. But I'll include it in the generic function anyway; I don't think it can cause any harm.